.jshintrc

```diff
@@ -29,5 +29,8 @@
     "maxparams": 3,
 
     // Limit the nesting to three levels for callbacks.
-    "maxdepth": 3
+    "maxdepth": 3,
+
+    // Identify possibly unsafe line breaks.
+    "laxbreak": true
```

Comment:
I think the [code I modified in the other PR](https://github.com/juwai/juwai-com-v2/pull/55/files#diff-7e043b55c2ac59a1ff81767e9f4dc9f0L152) should trigger the error.